script.js

```diff
@@ -0,0 +1,38 @@
+const userName = prompt("Яке Ваше імʼя?");
+const userSurname = prompt("Яке Ваше прізвище?");
+const userFavNum = +prompt("Яке Ваше улюблене число?");
+console.log(
+  `Вітаю, ${userName} ${userSurname}! Ваше улюблене число - ${userFavNum}`
+);
+
+const firstNum = +prompt("Введіть перше число");
+const secondNum = +prompt("Введіть друге число");
+const tot = firstNum + secondNum;
+const subtraction = firstNum - secondNum;
+const multiplication = firstNum * secondNum;
+const degree = firstNum / secondNum;
+console.log(
+  `Результати розрахунків!
+Сума: ${firstNum} + ${secondNum} = ${tot}
+Різниця: ${firstNum} - ${secondNum} = ${subtraction}
+Добуток: ${firstNum} * ${secondNum} = ${multiplication}
+Частка: ${firstNum} / ${secondNum} = ${degree}
+`
+);
+
+const UserDateOfBirth = +prompt("В якому році Ви народились?");
+const UserAge = 2022 - UserDateOfBirth;
```

Comment:
Its better to use `curentData.getFullYear()` for the current year calculation or at least initialize value with a constant to avoid "magic" values in the code.